pkg/data/components/TraceConverter.yaml

```diff
@@ -54,10 +55,11 @@ properties:
       configured. This properties supports sending a map of header keys and
       values.
     type: map
-  - name: Insecure
-    summary: Provide a way to disable TLS export.
+  - name: Secure
```

Comment:
@kentquirk offered a good alternative name for the option: `UseTLS`. It's like the no-judgment name for this setting.